src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private String[] color = {"green", "blue", "brown", "purple", "yellow"};
```

Comment:
The array of colors should be a constant. Consider declaring it as 'private static final' and using uppercase letters with underscores for the name to follow Java naming conventions for constants.